config/application.rb

```diff
@@ -17,7 +18,8 @@
 
 module Upaya
   class Application < Rails::Application
-    AppConfig.setup(YAML.safe_load(File.read(Rails.root.join('config', 'application.yml'))))
+    configuration = AppConfigReader.new.read_configuration
+    AppConfig.setup(configuration)
```

Comment:
could we consider writing out a copy of the resulting config back to the filesystem? I'd like the ability to inspect what config a box has, but if we don't write it out to disk, new values in S3 could override what the existing running processes actually have? Maybe we can set appropriate perms on the file so it's not world-readable?

Reply:
Just added something for that